main/src/com/google/refine/exporters/CustomizableTabularExporterUtilities.java

```diff
@@ -234,16 +240,31 @@ private enum DateFormatMode {
         @JsonProperty("custom")
         CUSTOM
     }
-    
+
     static private class ReconSettings {
         @JsonProperty("output")
         ReconOutputMode outputMode = ReconOutputMode.ENTITY_NAME;
         @JsonProperty("blankUnmatchedCells")
         boolean blankUnmatchedCells = false;
         @JsonProperty("linkToEntityPages")
         boolean linkToEntityPages = true;
+
+        ReconSettings(){}
```

Comment:
This should not be needed, I think: the default constructor should exist by default, no?

Reply:
I add other constructor, I think....